LICENSE

```diff
@@ -1,6 +1,11 @@
 MIT License
 
-Copyright (c) 2023 yasahi-hpc
+Copyright (c) 2023-2024 Centre national de la recherche scientifique (CNRS)
+Copyright (c) 2023-2024 Commissariat a l'énergie atomique et aux énergies alternatives (CEA)
+Copyright (c) 2023-2024 Yuuichi Asahi
+Copyright (c) 2023-2024 Paul Zehner
```

Comment:
You should be in the AUTHORS file indeed, but you can only claim copyright if you worked on the project as a personal project, outside of work. I let each of you @yasahi-hpc & @pzehner determine if that's the case.

Reply:
OK, I will remove our names